app/social/components/sexualPartners.html

```diff
@@ -0,0 +1,48 @@
+<div class="marital-partners">
+
+  <div class="alert alert-info" ng-if="!vm.checkedIn">
+    {{'CHECKIN_NOT_CHECKED_IN_TODAY'|translate}}
+  </div>
```

Comment:
Better if we show consistency here:
![image](https://user-images.githubusercontent.com/3810265/38796895-1ff11736-415d-11e8-8782-8b56d4080fc3.png)